via/views/view_pdf.py

```diff
@@ -51,6 +55,28 @@ def view_pdf(context, request):
     }
 
 
+@view.view_config(
+    route_name="python_proxy_pdf",
+    decorator=(has_secure_url_token,),
+)
+def python_proxy_pdf(context, request):
```

Comment:
This is `python_proxy_pdf`, it doesn't do anything related to OneDrive.

If we eventually have to write some OneDrive specific view it can go to its own module.

If we ever need to proxy any other PDFs, eg all pdfs coming from "example.com" we can just match the URL and take it to this view.